config.toml

```diff
@@ -93,8 +93,8 @@ weight = 1
 section = ["HTML", "print", "RSS"]
 
 [params]
-copyright = "The Kiali Project"
-privacy_policy = "https://policies.google.com/privacy"
+#copyright = "The Kiali Project"
+#privacy_policy = "https://policies.google.com/privacy"
```

Comment:
We probably need valid settings for these.  I'm not sure what they are but we should at least have a copyright, I would think.

Reply:
I chose to remove the copyright notice, because apparently, we are unsure who is. That is what I understand from this discussion that a community user opened: https://github.com/kiali/kiali/discussions/4252. No concrete answer was provided, so I prefer to remove the copyright notice until we are certain.

And about the privacy policy, do we have one?